stencil.config.ts

```diff
@@ -116,13 +116,9 @@ export const create: () => Config = () => ({
     name: "calcite-hydrated"
   },
   extras: {
-    appendChildSlotFix: true,
-    cssVarsShim: true,
-    dynamicImportShim: true,
-    safari10: true,
-    scriptDataOpts: true,
-    shadowDomShim: true,
-    slotChildNodesFix: true
+    dynamicImportShim: true, // do we need this for FF? https://github.com/Esri/calcite-components/issues/935
```

Comment:
@jcfranco do we need any of these 'extras'?

Reply:
> do we need this for FF

FYI, I didn't see any reference to Firefox in the linked issue.

Reply:
Lets decide which of these we need. https://stenciljs.com/docs/config-extras